src/scores/continuous/standard_impl.py

```diff
@@ -354,11 +354,10 @@ def pbias(
     """
     Calculates the percent bias, which is the ratio of the additive bias to the mean observed value, multiplied by 100.
 
-    Percent bias is used for evaluating and comparing forecast accuracy across stations or dataset with varying magnitudes.
-    By expressing the error as a percentage of the observed value, it allows for standardized comparisons, enabling assessment
-    of forecast performance regardless of the absolute scale of values. Like multiplicative_bias, Percent bias will return a np.inf
-    where the mean of `obs` across the dims to be reduced is 0.
-    It is defined as
+    Percent bias is used for evaluating and comparing forecast accuracy across stations or datasets with varying magnitudes.
+    By expressing the error as a percentage of the observed value, it allows for standardised comparisons, enabling assessment
+    of forecast performance regardless of the absolute scale of values. Like :py:func:`scores.continuous.multiplicative_bias`,
+    ``pbias`` will return a np.inf where the mean of ``obs`` across the dims to be reduced is 0. It is defined as
```

Comment:
You had `obs` (single set of backticks). When the API docstrings are rendered in readthedocs, one set of backticks causes the text to be rendered in italics. Two sets of backticks causes it to appear  in a grey box (for want of a better descriptor) - is that what you were trying for?

You can see what obs with two backticks on either side looks like here: https://scores.readthedocs.io/en/228-documentation-testing-branch/api.html#scores.continuous.pbias

You can see what obs with one backtick on either side looks like here: https://scores.readthedocs.io/en/latest/api.html#scores.continuous.pbias

Please note: this specific backtick rendering only applies to API docstrings rendering in readthedocs (it does not apply to non-API docstrings rendering in readthedocs). It took us a while to figure this out, and we still have docstrings we need to go and fix this in!

Reply:
Two backticks is right here.